agent.go

```diff
@@ -385,6 +385,100 @@ func (c *controller) agentClose() {
 	agent.networkDB.Close()
 }
 
+type Task struct {
+	Name       string
+	EndpointID string
+	EndpointIP string
+	Info       string
+}
+
+type ServiceInfo struct {
+	VIP   string
+	Tasks []Task
+	Ports []string
+}
+type epRecord struct {
+	ep   EndpointRecord
+	info string
+}
+
+func (n *network) Services() map[string]ServiceInfo {
+	eps := make(map[string]epRecord)
```

Comment:
This is just fine. My suggestion is to avoid nit picks on coding style/personal preference unless there is a clear violation of go idioms (which would be fixed by go fmt or caught by go vet).